src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -95,6 +95,19 @@ Robocopy "%PACKAGES_DIR%\MicroBuild.Core\%MICROBUILD_VERSION%\build\." "%TOOLRUN
 Robocopy "%PACKAGES_DIR%\Microsoft.Net.Compilers\%ROSLYNCOMPILERS_VERSION%\." "%TOOLRUNTIME_DIR%\net46\roslyn\." /E
 
 @echo on
```

Comment:
I feel like we need to make this optional in some way because not all consumers of BuildTools require ilasm. I wonder if we should hook this initialization up as part of https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/IL.targets, so it is pay-for-play.

Reply:
You mean lazily downloading it when something ends up needing it? How does it gel with "build from source" effort and building without internet connection?

Reply:
They will either have the ilasm package in their cache or they won't. If they don't then it will fail. While I don't mind this being in init-tools I want to make sure it is optional in some way and not required for every repo using BuildTools which is what you currently have.